satellite/rewards/partners_staticdb.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package rewards
+
+import (
+	"context"
+	"sort"
+
+	"github.com/zeebo/errs"
+)
+
+// PartnersStaticDB implements partner lookup based on a static definition.
+type PartnersStaticDB struct {
```

Comment:
Is it needed to tag this type with `Architecture: database`?

Reply:
The implementation is wired to the graph via `var _ PartnersDB = (*PartnersStaticDB)(nil)
`, but yeah, something like `architecture: database implementation` would probably make sense.

Reply:
I don't know that much what the tooling requires, I'm sure that you know. I raised the point in case it was forgotten.

Reply:
https://github.com/storj/storj/pull/3495